app/controllers/application_controller.rb

```diff
@@ -11,18 +11,19 @@ class ApplicationController < ActionController::API
 private
 
   def authorise_read!
-    if token.nil?
-      render_bad_token
-    elsif token.has_scope?("read") == false
-      render_forbidden "You need the scope 'read' to use this endpoint"
-    end
+    authorize_for!(READ_ROLE)
   end
 
   def authorise_write!
+    authorize_for!(WRITE_ROLE)
```

Comment:
It'd be nice to be consistent with British English vs American English spellings.

Let's settle on either `authorise` or `authorize` and stick to it, rather than using both spellings.